### ISSUE TITLE
Any chance on getting GCP support?

### ISSUE THREAD

**Issue by sroye98** [zdenham/next-static-utils#1]
This library sounds amazing and is in the nick of time. Just one question, do you plan on supporting GCP anytime soon?

**Comment by zdenham** [comment]
@sroye98 I can look into it. Curious if there is an edge function with GCP CDN similar to cloudfront edge functions.

**Comment by sroye98** [comment]
@zdenham Yes I believe they do have edge functions in their CDN service. Although I am not sure if we can access the functions on GCP like we can on AWS.
